Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ruby:3.2.2-alpine AS builder
 RUN apk update && apk upgrade && apk add --update --no-cache \
   build-base \
   curl-dev \
-  nodejs \
+  'nodejs=~20' \
```

Comment:
I checked a PR from 2 days ago, the docker logs say: `Installing nodejs (20.15.1-r0)`, which is the latest available in alpine. It's the same after this change. I guess it defaults to the active LTS. So this is redundant now, but will be useful when node 22 LTS hits active status later this year.  It won't get bumped up unexpectedly.

Reply:
cool!